spec/feature_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'takeaway'
+
+describe 'User Stories' do 
+
+  it 'So I can check if I want to order something, I would like to see a list of dishes'do
+  takeaway = Takeaway.new
+  pretty_menu =takeaway.list_menu
+  expect{ takeaway.list_menu }.to output(pretty_menu).to_stdout 
+  end
+
+  it 'So I can the meal I want, I would like select some number of dishes available' do
```

Comment:
Metrics/LineLength: Line is too long. [87/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)